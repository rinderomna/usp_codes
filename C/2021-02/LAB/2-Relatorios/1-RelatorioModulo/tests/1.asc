25
-73941 -66320 -62730 -55268 -38350 -32227 -29701 -13398 -12633 -10649 -2163 12751 15738 19576 23148 31855 34113 39922 41194 42181 48980 57155 68881 71613 74957
