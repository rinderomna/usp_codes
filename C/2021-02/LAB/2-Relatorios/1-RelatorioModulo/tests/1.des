25
74957 71613 68881 57155 48980 42181 41194 39922 34113 31855 23148 19576 15738 12751 -2163 -10649 -12633 -13398 -29701 -32227 -38350 -55268 -62730 -66320 -73941
