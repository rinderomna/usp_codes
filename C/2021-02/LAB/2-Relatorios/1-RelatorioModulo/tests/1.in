25
-35186 28399 -43672 -9192 -52334 -83755 1225 66484 -41221 55823 -14252 -89558 10957 -84050 83580 12686 -52014 59057 -40674 -89167 82306 -19513 41171 91161 -16900
