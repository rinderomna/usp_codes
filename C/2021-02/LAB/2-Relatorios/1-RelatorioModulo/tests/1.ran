25
-12633 -32227 -38350 -73941 -62730 48980 74957 42181 -2163 31855 41194 12751 71613 19576 -13398 39922 -10649 -55268 68881 57155 -66320 15738 34113 -29701 23148
