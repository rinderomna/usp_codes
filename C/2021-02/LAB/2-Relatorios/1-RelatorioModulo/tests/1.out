-89558 -89167 -84050 -83755 -52334 -52014 -43672 -41221 -40674 -35186 -19513 -16900 -14252 -9192 1225 10957 12686 28399 41171 55823 59057 66484 82306 83580 91161
